bundle/config/mutator/select_target.go

```diff
@@ -30,7 +32,7 @@ func (m *selectTarget) Apply(_ context.Context, b *bundle.Bundle) error {
 	// Get specified target
 	target, ok := b.Config.Targets[m.name]
 	if !ok {
-		return fmt.Errorf("%s: no such target", m.name)
+		return fmt.Errorf("%s: no such target. Available targets: %s", m.name, strings.Join(maps.Keys(b.Config.Targets), ", "))
```

Comment:
Even though I know this trips up the linter, it would be nice to have a trailing `.` here.